packages/grafana-eslint-rules/rules/no-untranslated-strings.cjs

```diff
@@ -17,31 +24,77 @@ const noUntranslatedStrings = createRule({
 
         const isUntranslatedProp =
           (node.value.type === 'Literal' && node.value.value !== '') ||
-          (node.value.type === 'JSXExpressionContainer' &&
+          (node.value.type === AST_NODE_TYPES.JSXExpressionContainer &&
             (node.value.expression.type === 'Literal' || node.value.expression.type === 'TemplateLiteral'));
 
         if (isUntranslatedProp) {
+          const errorCanBeFixed = canBeFixed(node, context);
```

Comment:
Can we only call `canBeFixed` if the path `shouldFix`? I'm sure it's not _too_ expensive, but seeing as this runs on every JSX attribute I think it's good to be as mindful as possible.

Reply:
ahh - nevermind. I see we need `errorCanBeFixed` for the suggestion. I'm blind.

Reply:
No problem - I've still refactored to have "should this be fixed" and "can this be fixed", and I moved the options checking into the utils